CHANGELOG.md

```diff
@@ -12,6 +12,20 @@
   - Add "nieuw" and "toegevoegd" labels and filters to concepts
   - Add support for concept archiving
 ### deploy instructions
+#### Virtuoso update
+  - stop the stack: `drc stop`
+  - start the "maintenance" frontend?
+  - start the virtuoso container: `drc start virtuoso`
+  - create a new checkpoint so the transaction log is cleared:
+      - enter the isql-v environment: `drc exec virtuoso isql-v`
+      - create a checkpoint: `exec('checkpoint');`
+      - exit the isql-v environment: `exit;`
+  - stop the virtuoso service
+  - make a backup of the data/db folder in case something goes wrong and we need to downgrade again `cp -r data/db some-backup-folder`
+  - remove the transaction log file: `rm data/db/virtuoso.trx`
+  - `drc up -d virtuoso` to update to the new image
+  - check the logs to see if nothing strange happened
```

Comment:
Not sure if this is suited for the hosted environments, but this is what I did locally and it seemed to work.